plugins/@grouparoo/bigquery/src/lib/table-import/getPropertyValues.ts

```diff
@@ -90,7 +90,11 @@ export const getPropertyValues: GetPropertyValuesMethod = async ({
     types
   );
   if (addAnd) query += ` AND`;
-  query += ` ${inClause}`;
+  // for BigQuery we need to use UNNEST: `id in UNNEST(1,2,3)`
+  // See https://github.com/googleapis/nodejs-bigquery/blob/master/samples/queryParamsPositionalTypes.js#L37
+  query += ` ${inClause}`.replace(" IN (", " IN UNNEST (");
```

Comment:
it feels like  this change in be `filterOperation` where the type is `FilterOperation.In` - would that work?

Reply:
Oh! Great idea! Thanks!
https://github.com/grouparoo/grouparoo/pull/979